scm/test_helper.go

```diff
@@ -2,24 +2,30 @@ package scm
 
 import (
 	"context"
+	"testing"
 
 	"github.com/beatlabs/github-auth/app"
+	"github.com/quickfeed/quickfeed/internal/env"
 )
 
-func TestSCMManager() *Manager {
+// TestSCMManager sets the current provider to "fake", creates a "test" organization
```

Comment:
"fake" seems to be "test" now. So the comment is wrong. But should we use `test-name` and `test-path` or something simpler to signal which fields the variable is referring to? With `"test"` in both values it may be easy to mix things up.

Reply:
But it calls `env.SetFakeProvider` which sets `provider = 'fake'`, or am I missing something?

I am currently a bit confused about the `Path` field in organization-related things. Looks like it can be safely renamed to `Name` in the `Organization` type. And in the `OrganizationOptions` it is only used by GitLab's `CreateOrganization` method and we can probably just ignore this field in test helpers and fake scm.